.github/workflows/remove-all-security-groups.yml

```diff
@@ -0,0 +1,27 @@
+name: Remove All Security Groups
+
+on:
+    workflow_call:
+    workflow_dispatch:
+
+jobs:
+    remove-all-hackathon-participants:
```

Comment:
Please update the job name as it doesnt reflect the activity

Reply:
done